integration_tests/cairozero_test.go

```diff
@@ -62,7 +69,12 @@ func TestCairoZeroFiles(t *testing.T) {
 			continue
 		}
 
-		path := filepath.Join(root, dirEntry.Name())
+		var path string
+		if contains(testFiles1, dirEntry) {
```

Comment:
This is becoming of quadratic complexity. Can you make it linear using a map ?

Reply:
done